docs/usage.rst

```diff
@@ -289,3 +289,14 @@ deduplicate by taking into account the URL:
 .. sentry:edition:: hosted, on-premise
 
     For more information, see :ref:`custom-grouping`.
+
+Preventing Abuse
```

Comment:
Maybe explain what kind of abuse is at stake here? And explain that we use the Referer/Origin header to validate. Without setting this, someone could embed your DSN on their site and report exceptions to your account, etc.